graphql.go

```diff
@@ -111,8 +111,8 @@ func Logger(logger log.Logger) SchemaOpt {
 // Response represents a typical response of a GraphQL server. It may be encoded to JSON directly or
 // it may be further processed to a custom response type, for example to include custom error data.
```

Comment:
Can you, please, add this link to the comment: https://github.com/facebook/graphql/pull/384/commits/7b40390d48680b15cb93e02d46ac5eb249689876

Reply:
Done.